src/Mutant/MetricsCalculator.php

```diff
@@ -134,7 +134,7 @@ public function getMutationScoreIndicator(): float
         $defeatedTotal = $this->killedCount + $this->timedOutCount + $this->errorCount;
 
         if ($this->totalMutantsCount) {
-            $detectionRateAll = round(100 * ($defeatedTotal / $this->totalMutantsCount));
+            $detectionRateAll = floor(100 * ($defeatedTotal / $this->totalMutantsCount));
```

Comment:
I believe at least Covered MSI (line 167) should also be updated since we used it in `--min-coverered-msi` - the same bug will be here.

And what about `getCoverageRate()`? Probably having the same logic there would be consistent?

Reply:
Yes, I think that's going to be more proper to account for the worst case for all metrics.